templates/outer-loop/maven/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY --chown=1001:0 ./pom.xml /work/outer-loop-app/pom.xml
 RUN cd /work/outer-loop-app && \
     echo "QUICK START SECURITY IS NOT SECURE FOR PRODUCTION ENVIRONMENTS. IT IS BEING REMOVED" && \
     rm -f src/main/liberty/config/configDropins/defaults/quick-start-security.xml && \
-    mvn -e liberty:create package
+    mvn -e package liberty:create -Dliberty.runtime.version={{.OL_RUNTIME_VERSION}}
```

Comment:
Why did we switch the order here?

Reply:
no particular reason. It is just to be consistent with what we do everywhere else. The main/important change here is that i added the liberty runtime version to prevent downloading the most current version of the driver also.

Reply:
I think the old way is slightly more useful.. in case the 'package' phase had something bound to it that was say copying to a Liberty directory (and relying to the create to have been run).     This is a stack-invented pattern, not a well-known practice.. so I don't have the firmest ground to stand on, but still think it was better.

Yes, avoiding the download was a good idea.

Reply:
sure no problem. Will change that.